src/media/css/lib/mixins.styl

```diff
@@ -186,8 +186,25 @@ category-icon-position($slug, $scale=1) {
     background-position: ($positions[0] * $cat-icon-size-default * $scale * -1) ($positions[1] * $cat-icon-size-default * $scale * -1);
 }
 
+ltr() {
```

Comment:
Yeah, I try to do that also. I've been finding this helps make things a bit clearer though in some cases. I used it on the [search dropdown](https://github.com/mstriemer/fireplace/blob/3882d962d87305e05b5f672b9c04774384a6e904/src/media/css/elements--header.styl#L150-L161) and I think it looks pretty good.